index.html

```diff
@@ -2933,6 +2943,10 @@ <h2 class="code-line" data-line-start=9 data-line-end=10><a id="Companies_that_r
 			</tbody>
 		</table>
 		<p></p> 
+		<h2 class="code-line" data-line-start=2 data-line-end=3><a id="Notes_3"></a>Do not give up!</h2>
+		<p>Sure, Software Engineers are the most wanted IT specialists but from experience, companies are much more likely to seek candidates for QAs, PMs, Designers and other vacancies.</p>
```

Comment:
I think I've lost your thought a bit. Are you saying that other technical roles are more popular that software engineers?

Reply:
My bad, didn't compose it right.